.github/CONTRIBUTING.md

```diff
@@ -1,12 +1,12 @@
 # Globe&Citizen Contributing Guide
 
-Hi! I'm really excited that you are interested in contributing to Vue.js. Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
+Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
 
 - [Code of Conduct](https://github.com/globe-and-citizen/Celebrity-Fanalyzer/.github/CODE_OF_CONDUCT.md)
-- [Issue Reporting Guidelines](#issue-reporting-guidelines)
-- [Pull Request Guidelines](#pull-request-guidelines)
-- [Development Setup](#development-setup)
-- [Project Structure](#project-structure)
+- [Issue Reporting Guidelines (link under dev)](#)
+- [Pull Request Guidelines (link under dev)](#)
+- [Development Setup (link under dev)](#)
+- [Project Structure (link under dev)](#)
```

Comment:
The links here point to the current document headings. Do you plan something?

Reply:
Eventually, "Yes". These documents will be edited repeatedly. I wanted to keep them as reminders of what content was necessary in future versions.